fpp

```diff
@@ -67,6 +68,8 @@ for opt in "$@"; do
     exit 0
   elif [ "$opt" == "--record" -o "$opt" == "-r" ]; then
     echo "Recording input and output..."
+  elif [ "$opt" == "--non-interactive" -o "$opt" == "-n" ]; then
```

Comment:
i feel like `-n` is a bit short here. `-ni`? just in case we need n for something later